lib/geokit/geocoders/google.rb

```diff
@@ -150,6 +150,9 @@ def self.single_json_to_geoloc(addr)
 
         set_bounds(loc, addr)
 
+        loc.place_id = addr['place_id']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.